src/components/Button/Button.tsx

```diff
@@ -138,6 +139,7 @@ export default class Button extends React.PureComponent<ButtonProps, {}> {
     color: ButtonColor.SECONDARY,
     iconPosition: ButtonIconPosition.LEFT,
     size: ButtonSize.REGULAR,
+    status: ButtonStatus.ENABLED,
```

Comment:
It's worth documenting that this is the default by adding a `@default` tag in the comment for the corresponding property.

Reply:
added